### DISCUSSION TITLE
Use localeDropdown in footer?

### DISCUSSION THREAD

**Discussion by gouku** [facebook/docusaurus#5957] [Q&A] [answered]
Is there any easy way to show `localeDropdown` in footer instead of the navbar?

When I put `type: 'localeDropdown'` in the footer I got the following error:

```
A validation error occurred.
The validation system was added recently to Docusaurus as an attempt to avoid user configuration errors.
We may have made some mistakes.
If you think your configuration is valid and should keep working, please open a bug report.

ValidationError: "footer.links[0].items[0]" must contain at least one of [to, href, html]
```

**Comment by Josh-Cena** [comment, marked as answer]
No, there isn't an easy way—navbar and footer are completely different, and we've never even designed a dropdown for footer. What's your desired UI/UX in this case?

**Comment by gouku** [comment]
@Josh-Cena It's inspired by Stripe:

![image](https://user-images.githubusercontent.com/9361308/142139869-88c60f7e-7ebd-4df4-8e05-0711f2940dd0.png)

Our navbar used up almost all space for normal links. It's better to have locale dropdown in footer or somewhere else.

**Reply by slorber** [reply]
Can you provide a link, please?

It's not really a column, it's more a list of locales

In general we want to allow the usage of custom navbar, sidebar, footer items, so you should be able to build this item yourself in the future. 

I don't think adding a pre-made item is worth it for this use-case, unless many users use it (it would only be a "shortcut" somehow)

I believe you can already do that today quite easily: just create an array of locales, and add a footer item for each locale in your array. You can easily compute the link target on your config file

**Reply by gouku** [reply]
You can simply access it at the footer of the Stripe homepage: https://stripe.com/

> I believe you can already do that today quite easily: just create an array of locales, and add a footer item for each locale in your array. You can easily compute the link target on your config file

Thanks. This seems a reasonable workaround for now.

**Reply by slorber** [reply]
I see, so it's a real dropdown here, not very usual IMHO.

Maybe add a request on our canny feature requests board and we'll see if enough users want this?
